courses/basic-blocks/steps/00_setup/en.md

```diff
@@ -5,7 +5,7 @@
 Before getting your hands dirty and learning more about VTEX IO's Store Framework, you'll need to set up a few basic configurations, such as:
 
 - Installing **Git**;
-- Installing **Toolbelt**;
+- Installing **VTEX IO's CLI**;
```

Comment:
The GitHub repository of [VTEX IO CLI](https://github.com/vtex/toolbelt) is still named toolbelt, so maybe it is a good idea to change (only this line) to "Installing VTEX IO's CLI (former VTEX toolbelt)" or something similar?

Reply:
Toolbelt is the name of the GitHub project of the VTEX IO CLI, but the external developer doesn't interact with the GitHub project and doesn't even need to know about it. The developer/documentation user is familiar with the concept of a CLI, so using "Toolbelt" just makes things more complicated.

Reply:
Got it, makes sense!

Reply:
Can you approve the PR so I can merge it? 🙊🙈